shell/components/SideNav.vue

```diff
@@ -31,7 +31,7 @@ export default {
   },
 
   created() {
-    this.queueUpdate = debounce(this.getGroups, 500);
+    this.queueUpdate = debounce(this.getGroups, 250);
```

Comment:
What makes this new value a reasonable value? It seems like we can't possibly know a reasonable value since I believe a reasonable value is influenced by latency which will vary from use case to use case.

It would be really nice if we could remove it or comment why this new value makes sense.

Reply:
I went back in history and the original debounce came in at 500 in https://github.com/rancher/dashboard/pull/355/files#diff-433a9abd4ca70520f69fc064c160bb093918ce26dda6087a8bbfdbe2895d1944R112 (layouts/default.vue created). There it's only used to stem changes from `counts` coming over socket, which is similar to how it currently works (stem all changes coming socket.

I sliced in half just because the side nav was more performant, but thinking about it we could just reduce the time to double the count. Will revert and put in a general comment for why it's there